### PR TITLE
Update info.xml

### PR DESCRIPTION
**Pull request by hurradieweltgehtunter**
updated screenshot and description

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 <info>
 	<id>market</id>
-	<name>Market</name>
-	<summary>ownCloud Marketplace integration</summary>
-	<description>Connect an ownCloud instance with the ownCloud Marketplace to browse, install and update ownCloud apps easily from within the ownCloud web interface. To use this application you have to be logged in as an administrator.</description>
+	<name>ownCloud MarketApp</name>
+	<summary>Integrate the ownCLoud marketplace into your ownCloud</summary>
+	<description>Easily manage ownCloud apps from within your ownCloud web interface. It connects your ownCloud with your marketplace account and lets you browse, install and update any apps from inside your ownCloud instance.
+
+Please note: Since ownCloud X (10.0) every instance gets shipped with this app included. You do not need to install it separately.
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: pointless - people use the marketplace to update apps - quite regular use case - should not be part of the app description

File: appinfo/info.xml [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+To use this application click on "Files" in the top left corner and click on "Market" (cart icon) (Administrator privileges required) </description>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: this is true for every app - pointless - please remove as well

File: appinfo/info.xml [hunk 3]
```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0"?>
 <info>
 	<id>market</id>
-	<name>Market</name>
-	<summary>ownCloud Marketplace integration</summary>
-	<description>Connect an ownCloud instance with the ownCloud Marketplace to browse, install and update ownCloud apps easily from within the ownCloud web interface. To use this application you have to be logged in as an administrator.</description>
+	<name>ownCloud MarketApp</name>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: It was agreed to not include 'ownCloud' in the app names - please revert this as well

File: appinfo/info.xml [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
+	<summary>Integrate the ownCLoud marketplace into your ownCloud</summary>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: typo L -> l

File: appinfo/info.xml (hunk 3 above)
Comment (DeepDiver1975) [inline appinfo/info.xml]: also App is a pointless name suffix - these are all apps

File: appinfo/info.xml (hunk 2 above)
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: true for every app but not false. Market app is quite hidden for users who never used it.

File: appinfo/info.xml (hunk 1 above)
Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: dont know what u mean by that and what u r referring exactly

Comment (hurradieweltgehtunter) [inline appinfo/info.xml]: You should not tell what people do, it should describe what users CAN do with this app. Updates are possible with the market app

**Comment by hurradieweltgehtunter** [comment]
Updated info.xml.
Please merge when travis is ready.

**Pull request merged**